spec/changelog.yml

```
@@ -10,6 +10,9 @@
     - description: Add support for semantic_text field definition.
       type: enhancement
       link: https://github.com/elastic/package-spec/pull/807
+    - description: Add support for Agentless resources.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/853
```

I think this could be added in a potential 3.3.2. The options added here would be ignored on current versions of kibana, right?
```suggestion
- version: 3.3.2-next
  changes:
    - description: Add support for Agentless resources.
      type: enhancement
      link: https://github.com/elastic/package-spec/pull/853
```